src/main/java/org/sonar/plugins/stash/PullRequestRef.java

```diff
@@ -1,23 +1,52 @@
 package org.sonar.plugins.stash;
 
-import com.google.auto.value.AutoValue;
+public class PullRequestRef {
+    private String project, repository;
+    private int pullRequestId;
 
-@AutoValue
-public abstract class PullRequestRef {
-    public abstract String project();
-    public abstract String repository();
-    public abstract int pullRequestId();
+    public String project() {
+        return project;
+    }
 
+    public String repository() {
+        return repository;
+    }
+
+    public int pullRequestId() {
+        return pullRequestId;
+    }
 
     public static Builder builder() {
-        return new AutoValue_PullRequestRef.Builder();
+        return new Builder();
+    }
+
+    private PullRequestRef(String project, String repository, int pullRequestId) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)